### ISSUE TITLE
Python 3.6 Environment installs Python 2.7

### ISSUE THREAD

**Issue by joesoap99** [ContinuumIO/anaconda-issues#1412]
Windows 8.1, 64 bit
Anaconda 4.30 (sometimes calls it 4.38??)
Root Python 2.7.13
Environment Python 3.51

My base installation is Python 2.7.13 (Root)
Using Navigator, I went through the steps of adding a Python 3.6 environment.
It installed, but what it installed was a Python 2.7 environment.

**Comment by goanpeca** [comment]
Will check!

**Comment by chdoig** [comment]
I haven't been able to reproduce from a fresh Anaconda 4.3.0 environment.

<img width="1440" alt="screen shot 2017-02-03 at 2 41 07 pm" src="https://cloud.githubusercontent.com/assets/4940549/22607847/558dde9c-ea1f-11e6-8927-0cc9670a0d78.png">

<img width="1440" alt="screen shot 2017-02-03 at 2 46 04 pm" src="https://cloud.githubusercontent.com/assets/4940549/22607898/8f3eb1de-ea1f-11e6-9ffc-42c7b485e205.png">

@joesoap99, did you start from a new anaconda installation, or did you ran: conda update anaconda?

Can you provide the output of running "conda info" and "conda list"? Thank you

**Comment by joesoap99** [comment]
No I upgraded an Anaconda 4.20 set up that worked fine.
I used a mix of conda update conda and Navigator to upgrade.
See my earlier recent threads on Rstudio when things first started to go wrong.

I attach the requested files for my Pyt
![1](https://cloud.githubusercontent.com/assets/8518350/22613807/60c81ee0-ea73-11e6-847b-2736f53e290a.png)
![2](https://cloud.githubusercontent.com/assets/8518350/22613813/6a454e70-ea73-11e6-860d-8a8c3bc064ef.png)
![3](https://cloud.githubusercontent.com/assets/8518350/22613816/6fcbfc90-ea73-11e6-99a2-800b0897bf73.png)
![4](https://cloud.githubusercontent.com/assets/8518350/22613820/750239c2-ea73-11e6-9240-a98a64661521.png)



hon 3.5 environment ( which is built on a Python 2.7 root)

**Comment by joesoap99** [comment]
I attach similar information for the python 2.7 Root

![1](https://cloud.githubusercontent.com/assets/8518350/22613962/dc04bf22-ea74-11e6-8ef6-2ece2acad3ed.png)
![2](https://cloud.githubusercontent.com/assets/8518350/22613965/e2212044-ea74-11e6-8111-d1b0758f32bf.png)
![3](https://cloud.githubusercontent.com/assets/8518350/22613966/e6a3e228-ea74-11e6-8613-37070eb41085.png)
![4](https://cloud.githubusercontent.com/assets/8518350/22613968/ea989838-ea74-11e6-908d-fc5fe00cc387.png)

**Comment by joesoap99** [comment]
I had already deleted the Python 3.6 environment which loaded as full Python 2.7 environment instead.
I rebuilt with the same result but dleted that too.
so I built another Python 3.6 environment and this time it did have Python 3.6 but it only had 5 packages total in it.


![1](https://cloud.githubusercontent.com/assets/8518350/22614126/75a21f84-ea76-11e6-9973-ffb33879e19e.png)

**Comment by joesoap99** [comment]
Regarding Python 3.6 environment appearing to only install 5 packages.
I fixed this by accident.
Updating the list still gave 5 elements.

I installed Spyder from the Navigator icon.
I closed Navigator and reopened, and the Python 3.6 environment showed the complete list of files installed.

The Windows Start menus for Jupyter Notebook no longer work but they do if you run jupyter notebook from the Anaconda 2 prompt or from python 2. The targets sets by the Win Start Menus no longer link up. The configuration files for the system to find files seem not to be robust enough for upgrades.

Could the Anaconda installation make it easier to set up a dedicated folders for Python2 and Python 3 Jupyter ipyn files? They are all mixed in by default with non-relevant files in my c:\users\name folder and the Python 2 notebooks are mixed up with the Python 3 notebooks.
